test/e2e/diagnose.go

```diff
@@ -82,7 +84,6 @@ func checkUnschedulablePods(clients *test.Clients, logger *logging.BaseLogger) {
 		}
```

Comment:
Above here there are some strings that look like they can be replaced by constants from `vendor/k8s.io/api/core/v1/types.go`. 

Constants are:
PodScheduled
ConditionFalse
PodReasonUnschedulable

Reply:
Done.